.kokoro/publish-docs.sh

```diff
@@ -19,15 +17,25 @@ python3.6 -m nox --version
 # build docs
 nox -s docs
 
-cd docs/_build/html
-
 python3 -m pip install gcp-docuploader
 
+# install a json parser
+sudo apt-get update
+sudo apt-get -y install software-properties-common
+sudo add-apt-repository universe
+sudo apt-get -y install jq
+
 # create metadata
 python3 -m docuploader create-metadata \
---name ${NAME} \
---version ${VERSION} \
---language python
+  --name=$(jq --raw-output '.name // empty' .repo-metadata.json) \
```

Comment:
I'm unsure what is wanted here:  `python3 setup.py --name` would give the "Python project" name, e.g. `google-cloud-foo`.

Reply:
This 'name' gets used to construct the url at our new docs site. It usually matches the project name, but sometimes it's different.

googleapis.dev/python/**bigquery**/latest

Reply:
Hmm, then maybe we should be using the one in `setup.py` for `--distribution-name`?

Reply:
Done.